warehousing-alternatives-compare-table.md

```diff
@@ -0,0 +1,18 @@
+## Differences between Cloudant to Db2 data flow solutions
+
+### Source options from Cloudant:
+
+technology | available document load endpoints | notes
+--- | --- | --- 
+Apache Bahir Spark connector | `/{db}/_changes`, `/{db}/_all_docs`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | -
+IBM Cloud Functions | `/{db}/_changes`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | - 
+Node-RED | `/{db}/_design/{design_doc}/_search/{search_index_name}` | -
+
+### Sink options to Db2:
+
+technology | load to Db2 notes
+--- | --- 
+Apache Bahir Spark connector | options available to create and overwrite Db2 table
```

Comment:
I think the available SQL operations for this would be `CREATE TABLE` & `INSERT`
The notes can be something like:
```
Spark save modes change table creation behavior:
`SaveMode.ErrorIfExists`  - try to create the table and error if it exists
`SaveMode.Append` - try to create the table, insert rows into existing table if present
`SaveMode.Overwrite` - try to create the table, dropping it first if it already exists
`SaveMode.Ignore` - try to create the table, doing nothing if it already exists
```